.env.template

```
@@ -178,7 +178,7 @@ DEVOPS_CHALLENGE_HOST=http://localhost:9010
 TEAM_ALLOCATION_HOST=http://localhost:8083
 SELF_TEAM_ALLOCATION_HOST=http://localhost:8084
 ASSESSMENT_HOST=http://localhost:8085
-TEMPLATE_HOST=http://localhost:8086
+EXAMPLE_HOST=http://localhost:8086
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXAMPLE_HOST key should go before the INTRO_COURSE_HOST key

(UnorderedKey)